exercises/practice/bank-account/.meta/example.lua

```diff
@@ -1,28 +1,36 @@
 local BankAccount = {}
 
+function BankAccount:open()
+  assert(not self._open)
+  self._open = true
+  self._balance = 0
+end
```

Comment:
Previously we didn't have an explicit `open` operation, but the tests make it clear that creating an account object must be independent from opening an account.